docs/flow/config-language/expressions/_index.md

```diff
@@ -6,3 +6,24 @@ weight: 400
 ---
 
 # Expressions
+
+Expressions represent or compute values that can be assigned to attributes
+within a configuration.
+
+Basic expressions are literal values, like `"Hello, world!"` or `true`.
+Expressions may also do things like [refer to values][] exported by components,
+perform arithmetic, or [call functions][].
+
+Expressions can be used when configuring any component. As all component
+Arguments have an underlying [type][], River will type-check expressions before
+assigning the result to an attribute. 
+
+The only exception to this rule are some special reserved blocks like
+[`logging`][] which are used to configure the global behavior of Grafana Agent
+Flow and do not support expressions but can only use literal values.
```

Comment:
this isn't strictly true. `logging` can use expressions, it just can't refer to other components in its expressions.

Sorry, this reads like something I wrote; I was wrong :)

Reply:
You're right, I should have verified that. Removing this sentence.